frontend/src/pages/functional_components/NewExperimentFC.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2023 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import BusyButton from 'src/atoms/BusyButton';
+import Button from '@material-ui/core/Button';
+import Input from 'src/atoms/Input';
+import { V2beta1Experiment } from 'src/apisv2beta1/experiment';
+import { Apis } from 'src/lib/Apis';
+import { PageProps } from 'src/pages/Page';
+import { RoutePage, QUERY_PARAMS } from 'src/components/Router';
+import { URLParser } from 'src/lib/URLParser';
+import { classes, stylesheet } from 'typestyle';
+import { commonCss, padding, fontsize } from 'src/Css';
+import { errorToMessage } from 'src/lib/Utils';
+import { getLatestVersion } from 'src/pages/NewRunV2';
+import { useMutation } from 'react-query';
+
+const css = stylesheet({
+  errorMessage: {
+    color: 'red',
+  },
+  // TODO: move to Css.tsx and probably rename.
+  explanation: {
+    fontSize: fontsize.small,
+  },
+});
+
+interface ExperimentProps {
+  namespace?: string;
+}
+
+type NewExperimentFCProps = ExperimentProps & PageProps;
+
+export function NewExperimentFC(props: NewExperimentFCProps) {
+  const urlParser = new URLParser(props);
+  const { namespace, updateDialog, updateSnackbar, updateToolbar } = props;
+  const [description, setDescription] = useState<string>('');
+  const [experimentName, setExperimentName] = useState<string>('');
+  const [isbeingCreated, setIsBeingCreated] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const pipelineId = urlParser.get(QUERY_PARAMS.pipelineId);
+
+  useEffect(() => {
+    updateToolbar({
+      actions: {},
+      breadcrumbs: [{ displayName: 'Experiments', href: RoutePage.EXPERIMENTS }],
+      pageTitle: 'New experiment',
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Since no dependency is included, this useEffect will be called every time the page is rendered. I am guessing you need to depend on `props` or any change from the `createExperiment` call result.